orangecontrib/text/tests/test_bag_of_words.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+from unittest.mock import patch
+
+from orangecontrib.text.bagofowords import BagOfWords
+from orangecontrib.text.corpus import Corpus
+
+
+def progress_callback():
+    return
+
+
+def error_callback(*args):
+    return
+
+
+def mock_gensim_dictionary():
+    raise Exception
+
+
+class BagOfWordsTests(unittest.TestCase):
+
+    def setUp(self):
+        self.bow = BagOfWords(
+                progress_callback=progress_callback,
+                error_callback=error_callback,
+        )
+
+    def test_create_bow(self):
+        corpus = Corpus.from_file('deerwester')
+        bag_of_words = self.bow(corpus, use_tfidf=True)
+        self.assertIsNotNone(bag_of_words.X)
```

Comment:
Could you also check the shape here. Now `X` could also be a `dict` and this will pass.